### ISSUE TITLE
Typo in train.py?

### ISSUE THREAD

**Issue by Blinorot** [spkgyk/RTFS-Net#1]
Dear @spkgyk,

Thank you for publishing your implementation! I have found two differences from the original paper in the code:

1. In `train.py` in line 99 you have:
```
        "train": PITLossWrapper(pairwise_neg_snr, pit_from="pw_mtx"),
```
This means that you use SNR loss for training. But in the paper (Appendix I), the SI-SNR loss is used.

2. In `train.py` in line 86 you have:
```
        scheduler = ReduceLROnPlateau(optimizer=optimizer, factor=0.5, patience=10)
```
This means that scheduler have patience 10. But in the paper (Appendix I), patience is set to 5. (Btw, in config, it is set to 49 and not used)

Could you, please, say which loss and patience are required to reproduce the paper?

Thank you in advance!

**Comment by spkgyk** [comment]
The code produces the results seen in the paper. 

1: SNR loss is used for training, but an epoch is saved when the validation loss increases. The validation loss is SI-SNR. This was omitted in the paper.

2: The hard coded values in the code are correct. You may remove the superfluous lines in the config:
```
# Sche config
sche:
    patience: 49
    factor: 0.5
```
